src/test/java/redis/clients/jedis/tests/commands/SlowlogCommandsTest.java

```diff
@@ -36,6 +36,7 @@ public void tearDown() throws Exception {
 
   @Test
   public void slowlog() {
+    jedis.slowlogReset();
```

Comment:
If you think this is necessary, do this in a separate PR.

Reply:
👍

Reply:
done - removed this change from PR